error-issue/test/issue_builder.test.ts

```diff
@@ -116,6 +211,14 @@ describe('IssueBuilder', () => {
       expect(builder.bodyNotes).toContain(
         '**Possible assignees:** `@someone`, `@someoneelse`'
       );
+
+      builder.possibleAssignees = jest.fn().mockReturnValue([]);
```

Comment:
Don't mock methods in the file-under-test - change the conditions of the test so that it ends up generating this result "naturally"

Reply:
Done.